Description: on mipsel PTRACE_GET_THREAD_AREA is defined in asm/ptrace.h
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: no
Last-Update: 2020-03-17

--- a/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions.cc
+++ b/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions.cc
@@ -42,8 +42,12 @@
 // the one in Ubuntu 16.04 LTS) is missing PTRACE_GET_THREAD_AREA.
 // asm/ptrace-abi.h doesn't exist on arm32 and PTRACE_GET_THREAD_AREA isn't
 // defined on aarch64, so don't try to include this on those platforms.
+#if defined(__mips__)
+#include <asm/ptrace.h>
+#else
 #include <asm/ptrace-abi.h>
 #endif
+#endif
 #endif  // !OS_NACL_NONSFI
 
 #if defined(OS_ANDROID)
